scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
Wait, your description at the very top of this page says one has to install lang/gcc or lang/gcc6. Doesn't this make gcc available in the $PATH?

In other words - was this failing *with* or *without* the GCC installed from ports?